deploy/scripts/src/cri-orange/address.ts

```diff
@@ -90,6 +90,18 @@ export function address(): void {
         { isStatusCode302 }
       )
       // 02_AddCRICall
+      if (env.staticResources) {
```

Comment:
Can we add this to the `timeGroup` below too, as we want to capture the response time of the whole request